modules/network-observability-flowcollector-configuration-parameters.adoc

```diff
@@ -0,0 +1,98 @@
+* `ebpf.interfaces` and `ebpf.excludeInterfaces` specify the interface names from where flows are collected. You can reduce the overall observed traffic by restricting or excluding interfaces, which can help with resource management. If empty, the agent will fetch all the interfaces in the system, excepting the ones listed in ExcludeInterfaces. If an entry is enclosed by slashes (e.g. `/br-/`), it will match as regular expression, otherwise it will be matched as a case-sensitive string.
+* `ebpf.imagePullPolicy` is an enum value that specifies the Kubernetes pull policy for the defined image. The enum values are `IfNotPresent`, `Always`, `Never`. The default is `IfNotPresent`.
+* `ebpf.logLevel` defines logLevel for the eBPF agent. Possible values are `trace`, `debug`, `info`, `warn`, `error`, `fatal`, `panic`. The default is `info`.
+*`ebpf.privileged` is a boolean value that is set to `false` by default. The operator sets granular capabilities (BPF, PERFMON, NET_ADMIN, SYS_RESOURCE) to the container, to enable its correct operation. If for some reason these capabilities cannot be set, such as there is an old kernel version not knowing CAP_BPF, then you can turn on this mode for more global privileges.
+* `ebpf.debug.env` specifies custom environment variables to the Flowlogs-Pipeline pod. This field is useful for passing some concrete performance-tuning options, such as GOGC and
+GOMAXPROCS.
+
+|`spec.consolePlugin`
+|`object`
+a|Specifies configuration settings for preset filters that are displayed in the Console plugin.  
+
+* `consolePlugin.autoscaler` defines the specifications of a horizontal pod autoscaler to set up for the plugin Deployment.
+* `consolePlugin.imagePullPolicy` is the Kubernetes pull policy for the image defined.
+* `consolePlugin.logLevel` defines logLevel for the console plugin backend. Possible values are `trace`, `debug`, `info`, `warn`, `error`, `fatal`, `panic`. The default is `info`.
+* `consolePlugin.port` is the plugin service port. 
+* `consolePlugin.portNaming` defines the configuration of the port-to-service name translation.
+* `consolePlugin.quickFilters` defines preset configuration for Console's quick filters. The `filter` field defines a map of keys and values. You can name the filters in the `name`, which is displayed in the web console. The bang (!) character, placed at the end of a key, negates the key. Values are defined in a list as a comma-separated string. These settings offer a way to quickly apply different filters, such as showing or hiding the network of a pod, infrastructure network, or application network. They can be tuned to reflect the different workloads running on your cluster. You can define any number of filters, and set any of them being active by default. See the sample `FlowCollector` resource for more context.
```

Comment:
Since there is a dedicated section below maybe we should remove most of the description here and link to the below section ?